code/headers/mq_sensor.hpp

```diff
@@ -11,19 +11,14 @@ namespace r2d2::gas_detection {
     template <int AmountOfGasses>
     class mq_sensor_c : public gas_sensor_interface_c<AmountOfGasses> {
     private:
-        /**
-         * @param [rl] The load resistance on the board in kilo ohms.
-         * @param [ro] The value of resistance in gas concentration.
-         * @param [ro_clean_air_factor] The value of resistance in fresh air.
-         * This can be different with other MQ sensors.
-         * @return
-         */
         hwlib::target::pin_adc &adc_pin;
-        int rl = 10;
-        int ro = 0;
+        constexpr static int rl =
+            10;     /**< The load resistance on the board in kilo ohms. */
+        int ro = 0; /**< The value of resistance in gas concentration.r */
```

Comment:
I recommend changing this variable name to something more clear to the reader. two character variable names are hard to read and understand in my opinion. I have no clue what is store in this value when reading the code.